plugins/out_es/es.c

```diff
@@ -800,6 +800,11 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_elasticsearch, aws_region),
      "AWS Region of your Amazon ElasticSearch Service cluster"
     },
+    {
+     FLB_CONFIG_MAP_STR, "aws_sts_endpoint", "",
+     0, FLB_TRUE, offsetof(struct flb_elasticsearch, aws_sts_endpoint),
+     "STS endpoint"
```

Comment:
Description should be longer. Like: "Custom endpoint for the AWS STS API, used with the AWS_Role_ARN option"

Reply:
Also, since STS endpoint only makes sense if aws_role_arn is also specified, add a check for that. If aws_sts_endpoint is present without role_arn, print and error message